content/docs/command-reference/remote/add.md

```diff
@@ -199,17 +199,27 @@ $ dvc remote add myremote "azure://"
 
 ### Click for Google Drive
 
-Please check out
-[Setup a Google Drive DVC Remote](/doc/user-guide/setup-google-drive-remote) for
-a full guide on configuring Google Drives for use as DVC remote storage,
-including obtaining the necessary credentials, and how to form `gdrive://` URLs.
+To start using a GDrive remote, fist add it with a
+[valid URL format](/doc/user-guide/setup-google-drive-remote#url-format-to-specify-folder-location).
+Then simply use any DVC command that needs it (e.g. `dvc pull`, `dvc fetch`,
+`dvc push`), and follow the instructions to connect your Google Drive with DVC.
+For example:
 
 ```dvc
-$ dvc remote add -d myremote gdrive://root/path/to/folder
-$ dvc remote modify myremote gdrive_client_id <client ID>
-$ dvc remote modify myremote gdrive_client_secret <client secret>
+$ dvc remote add -d myremote gdrive://root/dvcstore
```

Comment:
> Now thinking a bit more, have some doubt about how we mention root as a good basic start.
From https://github.com/iterative/dvc.org/commit/ff380fc5c3e82faad03a85b6c70c841014a26a50#r38242840

Agree. Will review and also address https://github.com/iterative/dvc/issues/3586#issuecomment-608117910 ⏳

Reply:
Also addressing
> ... if we switch to something like gdrive://FOLDER_ID/path section that explains URLs becomes even more important.
> And explaining limitations for different URLs becomes very important (e.g. for grdive://root it won't be easy to share, and can be even dangerous multiple people will push data to two different GD accounts)
⏳

Reply:
OK I addressed all these concerns in 452924a7. Please check latest version 🙂